### ISSUE TITLE
About the pos weight

### ISSUE THREAD

**Issue by Eldo-rado** [ttanida/rgrg#32]
Hi, great work!

I noticed that the pos_weight in [binary_classifier_region_abnormal.py](https://github.com/ttanida/rgrg/blob/9520b6d7f555d458898c6795e57cc00f7f90182b/src/binary_classifier/binary_classifier_region_abnormal.py#L28) is set to 6.0 because normal regions are six times more prevalent than abnormal regions. My question is whether it would be more reasonable to set pos_weight as a vector, as the normal/abnormal ratio for the lungs is likely different from that of the diaphragm.

Additionally, is there any specific rationale behind choosing -1 for [logits > -1](https://github.com/ttanida/rgrg/blob/9520b6d7f555d458898c6795e57cc00f7f90182b/src/binary_classifier/binary_classifier_region_abnormal.py#L54) ?

Thanks!
